src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,26 @@
+
+
+const ArticleCard = (props) => {
+   const {title, topic, author, votes} = props;
+return ( <div className="Article_Card">
+    <p>Title: {title}</p>
+    <p>Topic: {topic}</p>
+    <p>Author: {author}</p>
+     <div>
+        <button>
+            Like
+        </button>
+     </div>
+     <p>Current likes: {votes}</p>
+     <div>
+        <button>
+            Dislike
+        </button>
+     </div>
```

Comment:
Appreciate you've not styled this yet but the `divs` wrapping your buttons are pointless. You can take em out 👍

Reply:
Thank you. Done